sql/index.go

```diff
@@ -285,7 +284,7 @@ func exprListsEqual(a, b []hash.Hash) bool {
 				continue
 			}
 
-			if reflect.DeepEqual(va.Sum(nil), vb.Sum(nil)) {
+			if reflect.DeepEqual(va, vb) {
```

Comment:
I think you can just do `va == vb`, since they are byte slices

Reply:
Hmm, check: https://play.golang.org/p/_dRDL0lErrV
I get an error: "(slice can only be compared to nil)"

Reply:
Sorry, the ones you can compare are byte **arrays**, not slices haha. Nvm, then